src/components/Tabs/index.tsx

```diff
@@ -5,21 +5,20 @@ import Button, { ButtonProps } from '../Button';
 import { useThemeClassName } from '../../hooks';
 import styles from './styles.css';
 
-type TabKey = string | undefined;
+type TabKey = string;
 
 export interface TabContextProps {
     activeTab: TabKey;
     setActiveTab: (key: TabKey) => void;
 }
 
 const TabContext = React.createContext<TabContextProps>({
-    activeTab: undefined,
+    activeTab: undefined as unknown as TabKey,
```

Comment:
This is wrong. It is clearly `undefined`, we should not cast it as `string` in any case.
Either, set default activeTab as `'default'` or set it as `string | undefined`

Reply:
I've set empty string as default value for now